org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/AEnv.java

```diff
@@ -477,17 +477,17 @@ public static void actionZoom(Lookup lookup, Object value)
 		zoomQuery.setZoomValue(value);
 		zoomQuery.addRestriction(column, MQuery.EQUAL, value);
 		zoomQuery.setRecordCount(1);    //  guess
-		if (zoomQuery.getZoomTableName() != null) {
+		if (zoomQuery != null && zoomQuery.getZoomTableName() != null) {
```

Comment:
It is impossible for zoomQuery to be null here.

Reply:
Adjusted